submissions/bilkisuTace/database/init.sql

```diff
@@ -0,0 +1,100 @@
+-- ----------------------------
+-- Table structure for products
+-- ----------------------------
+CREATE TABLE products (
+  product_id int(11) NOT NULL AUTO_INCREMENT,
+  name varchar(20) NOT NULL,
+  description varchar(250) NOT NULL,
+  image blob NOT NULL,
+  unit_price decimal(10,2) NOT NULL,
+  category_id int(10) NOT NULL,
+  admin_id int(20) unsigned NOT NULL,
```

Comment:
creating of `categories` and `admin` table need to come first before creating `products` table